Assets/_Game/Art Development/Redstonetech125/Animations/Purple Candle.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
I'm not sure why this is showing up here / whether it is safe to delete it or not